test/classes/initializers/generics/typeFunctions/fnReturnsGeneric.future

```diff
@@ -1,5 +1,9 @@
-bug: Can't write a type function that returns a generic class
+Feature Request: Allow type functions that return a generic class
```

Comment:
I'm not really convinced of the value of this particular future. Nonetheless I think you've updated it in a reasonable manner.